src/release/issue-triaging.md

```diff
@@ -54,6 +54,16 @@ To see a list of all labels, check out the "labels" page next to the search bar
 [IRLO]: https://internals.rust-lang.org/
 [Discord]: https://discord.gg/rust-lang
 
+### ICE Triage
+For [Issues that have both I-ICE and needs-triage](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AI-ICE%20label%3Aneeds-triage)
+* If it an older (like latest stable) version of rust, ask for or check the latest nightly (potentially check for duplicates)
+* If it does not have a reproduction, comment asking for one and add S-needs-repro. if there isn't one for around a month it should generally be closed.
+* If the reproduction is not minimal, add `E-needs-mcve` or create a Minimal Complete and Verifyable Example yourself.
+* Add `A-*` labels based on the code that causes the issue (check backtraces!),
```

Comment:
Suggestion: and maybe `T-compiler`/`T-types`/`T-rustdoc`/`WG-*`/`PG-*` as suitable?

Reply:
doesn't rustbot add T-compiler and I-Prioritize to all I-ICE issues?

Reply:
Hm, I'm not sure actually, but what I had in mind is that people don't always consistency use the I-ICE template.